spec/features/books/catalog_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe 'catalog', type: :feature do
+  describe 'filter' do
+    it 'cahnge filter' do
+      visit books_path
+      within 'main' do
+        expect(page).to have_content(I18n.t('filters.title_asc'))
+        find('.dropdown-toggle').click
```

Comment:
We need a driver for Capybara that supports JavaScript to test that dropdowns work correct